src/main/scala/cromwell/engine/backend/io/filesystem/gcs/GcsFileSystem.scala

```diff
@@ -29,16 +34,18 @@ object GcsFileSystem {
   */
 class GcsFileSystem private (gcsFileSystemProvider: GcsFileSystemProvider, gcsRoot: String) extends FileSystem {
 
+
+
   import GcsFileSystem._
 
-  val root = gcsRoot match {
-    case GsUriRegex(chunks) => new NioGcsPath(chunks.split(Separator), true)(this)
-    case _ => throw new InvalidPathException(gcsRoot, s"Root of GCS file system must be an absolute GCS path: $gcsRoot")
+  val root: Option[Path] = gcsRoot match {
```

Comment:
I agree with you, this root thing is bothering me too. What it gives you is the ability to call `toAbsolutePath` on a relative GCS path (all it really does is prepend the root to it).
However I'm unsure how useful that is since I removed the ability to create relative GCS paths from the `GcsFileSystem.getPath` method because it gets too confusing (it could successfully create a GCS path from "my/path" which there is no real way to know if it's a GCS or a Unix relative path..).
I'm gonna try and get rid of the root thing.